keyboards/ergodash/ergodash.h

```diff
@@ -3,8 +3,8 @@
 
 #include "quantum.h"
 
-#ifdef KEYBOARD_ergodash_rev1
```

Comment:
Is there a reason to remove the rev1 code here?

Reply:
No reason.
I will add as follows.

#ifdef KEYBOARD_ergodash_rev1
    #include "rev1.h"
#endif